app/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# HDE xml files installation
+set (WEARABLES_PROJECT_NAME Wearables)
+set (WEARABLES_XML_FILES xml/FTShoeLeftWearableDevice.xml
+                   xml/FTShoeRightWearableDevice.xml
+                   xml/FTShoesWearableDevice.xml
+                   xml/ICubWearableDevice.xml
+                   xml/SkinInsolesWearableDevice.xml
+                   xml/XsensSuitWearableDevice.xml
+
+                   xml/applications/WearableDevices.xml)
+
+install(FILES ${WEARABLES_XML_FILES}
+             DESTINATION ${CMAKE_INSTALL_PREFIX}/share/${WEARABLES_PROJECT_NAME}/)
```

Comment:
Since the project uses [`GNUInstallDirs`](https://github.com/robotology/wearables/blob/4ea2588b48cd1b21d32685eddaf14ba786f6a957/CMakeLists.txt#L11), you can use `${CMAKE_INSTALL_DATADIR}` instead of hardcoding `share`.

I think you can also specify a relative path by removing the `${CMAKE_INSTALL_PREFIX}` which is implicit.